### ISSUE TITLE
Ubuntu 64 bit crashes without error

### ISSUE THREAD

**Issue by jakehaas** [PinIdea/ASIC-X11-Miner#1]
Running ./cgminer on Ubuntu 64 bit asks for algorithm to be set (--x11)

so then running ./cgminer --x11 does not seem to do anything and returns me to a blank command line with no errors. Cannot start cgminer.

**Comment by whiskey1** [comment]
read the section about [usage](https://github.com/PinIdea/ASIC-X11-Miner/releases). it's not going to prompt you for device and will just quit without error, you've got to add -S /dev/ttyUSB0 or whatever device is.  i'd say working as is.

**Comment by soleo** [comment]
@jakehaas Could you upload the screenshot of your cgminer and the command you used for running ?

**Comment by jakehaas** [comment]
@whiskey1 I was adding the device to the command line ( -S /dev/ttyACM0).Sorry, I should have included my full command in the inital post.

After banging my head against the wall for hours, I finally figured it out... Even when running cgminer as root with sudo, I didn't have read/write access to /dev/ACM0 (or whaever the port is). I had to manually execute "chmod 777 /dev/ttyACM0" or add myself to the dialout group. Strange that It didnt work even as root, however...

@soleo Maybe it would be possible to add a warning saying something like "/dev/ttyACM0 : Permission denied" if cgminer cannot get the proper access it needs?
